resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -70,21 +70,24 @@
     private static final String LOCALHOST = "localhost";
     private static final InetAddress LOCALHOST_ADDRESS;
 
-    static final InternetProtocolFamily[] DEFAULT_RESOLVE_ADDRESS_TYPES = new InternetProtocolFamily[2];
+    static final InternetProtocolFamily[] DEFAULT_RESOLVE_ADDRESS_TYPES;
```

Comment:
@trustin I think the rule only checks the local scope, the suggestion from Sonarqube is that the elements are mutable even though the field is final and so its better to make it private if it is used only in the class or provide a getter that copies, but this is not applicable to this example